solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/impl/SolrFeature.java

```diff
@@ -246,7 +247,11 @@ public SolrFeatureScorer(FeatureWeight weight, Scorer solrScorer,
 
       @Override
       public float score() throws IOException {
-        return solrScorer.score();
+        try {
+          return solrScorer.score();
+        } catch (UnsupportedOperationException e) {
+          throw new FeatureException("Unable to extract feature during scoring");
```

Comment:
should pass the original exception in as well, so we dont lose the original's stack